R/quality_check.R

```diff
@@ -236,6 +254,52 @@ quality_check <- function(R_data,
                     cat(..1, sep="\n", "\n")
                   })',
     '```',
+    '\\newpage',
+    '# Verified records',
+    '',
+    '## Brood data',
+    '',
+    '```{r, echo=FALSE}',
+    '    purrr::pwalk(.l = Brood_approved_list,
+                 .f = ~{ cat(paste0("Record with BroodID ", ..2,
+                                    " and PopID ", ..1,
+                                    " has been verified by the data owner and does no longer violate check ",
```

Comment:
In theory it still violates the check...it's just that it doesn't matter anymore 😄 

```suggestion
                                    " has been verified by the data owner and is trustworthy",
```

Reply:
Sharp!

I still want to include the CheckID there, so what do you think of the following:
"Record with BroodID X and PopID Y violates check (CheckID Z) but has been verified by the data owner and is trustworthy."